skate-plugin/src/test/kotlin/com/slack/sgp/intellij/ChangeLogParserTest.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.sgp.intellij
+
+import com.google.common.truth.Truth.assertThat
+import java.time.LocalDate
+import org.junit.Test
+
+class ChangeLogParserTest {
+  // test with no entries and null changelogstring
+  @Test
+  fun testNoEntries() {
+    val (changeLogString, latestEntry) = ChangelogParser.readFile("", null)
+    assertThat(changeLogString).isNull()
+    assertThat(latestEntry).isEqualTo(LocalDate.now())
+  }
+
+  // test with one entry, no previous entry
+  @Test
+  fun testSingleEntryNullPreviousEntry() {
+    val input = "2023-06-28\nBug fixes\nNew features"
```

Comment:
nit: let's use raw strings for readability. Let's also make these as close to realistic as possible (i.e. with real formatting)

```suggestion
    val input = """
      2023-06-28
      ----------
      
      - Bug fixes
      - New features
    """.trimIndent()
```